kernel/uGraph.ml

```diff
@@ -20,7 +20,7 @@ module G = AcyclicGraph.Make(struct
     let compare = Level.compare
 
     let raw_pr = Level.raw_pr
-  end) [@@inlined] (* without inline, +1% ish on HoTT, compcert. See jenkins 594 vs 596 *)
```

Comment:
@SkySkimmer it seems that this annotation never did anything, do you think your measurement might have been withing noise ranges?

Reply:
Also, inlining a single initialization-time functor application probably cannot help much, right?

Reply:
It's possible